maltd-frontend/src/components/SearchBox/SearchBox.js

```diff
@@ -0,0 +1,34 @@
+/* eslint-disable react/jsx-filename-extension */
+import React from "react";
+import { Button } from "reactstrap";
+import PropTypes from "prop-types";
+import "./SearchBox.css";
+
+export default function SearchBox({
+  searchBox: { type, color, disabled, block, active, outline }
+}) {
+  return (
+    <Button
+      className="search-box"
+      type={type}
+      color={color}
+      disabled={disabled}
+      block={block}
+      active={active}
+      outline={outline}
+    >
+      Search
```

Comment:
this should be a prop that is passed into this component so that it can be rendered with different text dynamically for reuse - otherwise, this component will always render with the text "Search" which is not what we want 😄

Reply:
Great points! I'll make the corresponding changes now. 👍

Reply:
> this should be a prop that is passed into this component so that it can be rendered with different text dynamically for reuse - otherwise, this component will always render with the text "Search" which is not what we want 😄

Great points! I'll make the corresponding changes now. 👍